python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -251,7 +251,11 @@ def op_def(self) -> OpDefinition:
     @public
     @property
     def has_partition_key(self) -> bool:
-        """Whether the current run is a partitioned run."""
+        """Whether the current run is a partitioned run.
+
+        Returns:
+            bool:
```

Comment:
nit: I don't think we need the extra colon on these return types that we don't plan on annotating